tools/container/nvidia-toolkit/run.go

```diff
@@ -104,18 +115,26 @@ func main() {
 	log.Infof("Completed %v", c.Name)
 }
 
+func validateFlags(_ *cli.Context, o *options) error {
+	if filepath.Base(o.pidFile) != "toolkit.pid" {
```

Comment:
nit: Let's move `toolkit.pid` to a constant var

Reply:
Done.